hadoop-plugin/src/main/groovy/com/linkedin/gradle/scm/ScmPlugin.groovy

```diff
@@ -83,7 +76,16 @@ class ScmPlugin implements Plugin<Project> {
         }
       }
     }
+  }
 
+  @Override
+  void apply(Project project) {
+    applyMetadataTasks(project)
+    // Allows users to skip the plugin if they don't want the sources zip (performance improvement)
+    if (project.hasProperty("disableScmPlugin")) {
+      println("ScmPlugin disabled");
```

Comment:
Disabling the plugin before meant no sources zip and no buildMetadata.json file, now will mean no sources zip only.
Wondering if we need to be more specific in the message of this _println_.